srs_data/tagging.py

```diff
@@ -50,7 +67,7 @@ def _overwrite_uniform_sample_currently_in_s3(self):
     def _locally_update_corpora_tags(self):
```

Comment:
this should be called `_generate_corpora_tag_updates`. nothing is done locally, all it does is generate a directory that has a different file for each file in corpora, containing the tag updates that will be made to that file